source/overview/license-and-subscription.rst

```diff
@@ -71,9 +71,24 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
```

Comment:
```suggestion
You can renew your self-managed Mattermost Enterprise license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.

If you're a Mattermost Cloud customer, your subscription auto-renews so you don't need to follow this process.
```

Reply:
Out of interest, how do Mattermost Cloud customers change the parameters of their subscription? We mention that for self-managed and should mention it for Mattermost Cloud, for consistency.

Reply:
After they have upgraded to Cloud Professional there aren't any more options they can change to unless they want to cancel for which they currently have to contact support to do. Not sure if that answers your question.